internal/cmd/flink/command_iam_binding_create.go

```diff
@@ -0,0 +1,79 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newIamBindingCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a Flink IAM binding.",
+		Args:  cobra.NoArgs,
+		RunE:  c.iamBindingCreate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create Flink IAM binding for AWS us-west-2 and env-123.`,
+				Code: "confluent flink iam-binding create --cloud aws --region us-west-2 --environment env-123 --identity-pool pool-123",
+			},
+		),
+	}
+
+	c.addRegionFlag(cmd)
+	pcmd.AddCloudFlag(cmd)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	cmd.Flags().String("identity-pool", "", "Identity pool ID.")
```

Comment:
no flag for cloud and region? or inherited from other commands?

Reply:
Cloud and region flags are added in line 26 and 27